components/providers/provider-detail-client.tsx

```
@@ -6,12 +6,19 @@ import Link from "next/link";
 import { ResponsiveContainer } from "@/components/ui/responsive";
 import { Button } from "@/components/ui/button";
 import { Badge } from "@/components/ui/badge";
+import { Card } from "@/components/ui/card";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
 import { getFallbackImageUrl } from "@/lib/image-utils";
 import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogFooter, DialogTrigger } from "@/components/ui/dialog";
```

ESLint

[error] 'Dialog' is defined but never used.

(@typescript-eslint/no-unused-vars)